src/metadata.ts

```diff
@@ -106,6 +106,12 @@ export class MetadataPanel {
 
         const template = document.createElement('template');
         template.innerHTML = generateModal(this._guid, metadata);
+
+        // Stop progation when pressing a key (Jupyter widget)
+        (template.content.firstChild as HTMLElement).addEventListener('keydown', (event) => {
+            event.stopPropagation();
+        });
+
         this._modal = template.content.firstChild as HTMLElement;
         document.body.appendChild(this._modal);
```

Comment:
```suggestion
        this._modal = template.content.firstChild as HTMLElement;
        
        // Stop propagation of keydown events. This is required for the Jupyter integration, 
        // otherwise jupyter tries to interpret key press in the modal as its own input
        this._modal.addEventListener('keydown', (event) => {
            event.stopPropagation();
        });
        
        document.body.appendChild(this._modal);
```